controllers/update_status.go

```diff
@@ -175,23 +175,6 @@ func (s UpdateStatus) Reconcile(r *FoundationDBClusterReconciler, context ctx.Co
 		status.ProcessGroups = append(status.ProcessGroups, fdbtypes.NewProcessGroupStatus(processGroupID, internal.ProcessClassFromLabels(service.Labels), nil))
 	}
 
-	// Ensure that anything the user has explicitly chosen to remove is marked
-	// for removal.
-	for _, processGroupID := range cluster.Spec.InstancesToRemove {
-		for _, processGroup := range status.ProcessGroups {
-			if processGroup.ProcessGroupID == processGroupID {
-				processGroup.Remove = true
-			}
-		}
-	}
-	for _, processGroupID := range cluster.Spec.InstancesToRemoveWithoutExclusion {
```

Comment:
Moved into the validate instances method (I initially wondered why we didn't set `processGroup.Remove = true` here until I noticed he can save a few CPU cycles.